rewrite-java/src/main/java/org/openrewrite/java/format/MinimumViableSpacingVisitor.java

```diff
@@ -97,6 +97,14 @@ public J.ClassDeclaration visitClassDeclaration(J.ClassDeclaration classDecl, P
         return c;
     }
 
+    @Override
+    public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, P p) {
+        if (method.getName().getComments().isEmpty() && method.getName().getPrefix().getWhitespace().contains("\n")) {
+            return method.withName(method.getName().withPrefix(Space.EMPTY));
+        }
+        return super.visitMethodInvocation(method, p);
```

Comment:
A method invocation meeting this condition could be contained within another method invocation which meets this condition. In that circumstance the inner method would not be reformatted as no traversal into sub-elements happens when this condition is matched.